src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs-mirroring/cephfs-mirroring-list/cephfs-mirroring-list.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, ViewChild, OnInit } from '@angular/core';
+import { NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { BehaviorSubject } from 'rxjs';
+import { TableComponent } from '~/app/shared/datatable/table/table.component';
+import { CdTableAction } from '~/app/shared/models/cd-table-action';
+import { CdTableColumn } from '~/app/shared/models/cd-table-column';
+import { CdTableFetchDataContext } from '~/app/shared/models/cd-table-fetch-data-context';
+import { CdTableSelection } from '~/app/shared/models/cd-table-selection';
+import { URLBuilderService } from '~/app/shared/services/url-builder.service';
+import { ActionLabelsI18n } from '~/app/shared/constants/app.constants';
+import { ClusterRow, Cluster } from '../cephfs-mirroring.model';
+
+const BASE_URL = 'cephfs/mirroring';
+
+@Component({
+  selector: 'cd-cephfs-mirroring-list',
+  templateUrl: './cephfs-mirroring-list.component.html',
+  styleUrls: ['./cephfs-mirroring-list.component.scss'],
+  providers: [{ provide: URLBuilderService, useValue: new URLBuilderService(BASE_URL) }]
+})
+export class CephfsMirroringListComponent implements OnInit {
+  @ViewChild('table', { static: true }) table: TableComponent;
+  columns: CdTableColumn[];
+  tableActions: CdTableAction[];
+  context: CdTableFetchDataContext;
+  selection = new CdTableSelection();
+  modalRef: NgbModalRef;
+  subject$ = new BehaviorSubject<ClusterRow[]>([]);
+  mirroringData$ = this.subject$.asObservable();
+
+  dummyData = {
+    '/dnyaneetest': {
+      state: 'idle',
+      last_synced_snap: {
+        id: 6,
+        name: 'snap1',
+        sync_duration: 0,
+        sync_time_stamp: '256640.291449s',
+        sync_bytes: 0
+      },
+      snaps_synced: 1,
+      snaps_deleted: 0,
+      snaps_renamed: 0
+    }
+  };
+
+  constructor(public actionLabels: ActionLabelsI18n) {}
+
+  ngOnInit() {
+    this.columns = [
+      { name: $localize`Snapshot / Directory`, prop: 'snapshot', flexGrow: 2 },
```

Comment:
Woudn't this be subvolum / directory?

```suggestion
      { name: $localize`Subvolume / Directory`, prop: 'snapshot', flexGrow: 2 },
```
Also I think we should add truncation here